_includes/pbc/all/install-features/202311.0/marketplace/install-the-marketplace-merchant-portal-product-offer-service-points-feature.md

```diff
@@ -0,0 +1,178 @@
+
+Make sure the following modules have been installed:
+
+| MODULE | EXPECTED DIRECTORY |
+|-|-|
+| ProductOfferServicePointMerchantPortalGui | spryker/product-offer-service-point-merchant-portal-gui |
+
+{% endinfo_block %}
+
+## 2) Set up configuration
+
+Deny Back Office access for Back Office users to the ProductOfferServicePointMerchantPortalGui module.
+
+**src/Pyz/Zed/Acl/AclConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Acl;
+
+use Spryker\Shared\Acl\AclConstants;
+use Spryker\Zed\Acl\AclConfig as SprykerAclConfig;
+
+class AclConfig extends SprykerAclConfig
+{
+    /**
+     * @return array<array<string, mixed>>
+     */
+    public function getInstallerRules(): array
+    {
+        $installerRules = parent::getInstallerRules();
+        $installerRules = $this->addMerchantPortalInstallerRules($installerRules);
+
+        return $installerRules;
+    }
+
+    /**
+     * @param array<array<string, mixed>> $installerRules
+     *
+     * @return array<array<string, mixed>>
+     */
+    protected function addMerchantPortalInstallerRules(array $installerRules): array
+    {
+        $bundleNames = [
+            'product-offer-service-point-merchant-portal-gui',
+        ];
+
+        foreach ($bundleNames as $bundleName) {
+            $installerRules[] = [
+                'bundle' => $bundleName,
+                'controller' => AclConstants::VALIDATOR_WILDCARD,
+                'action' => AclConstants::VALIDATOR_WILDCARD,
+                'type' => static::RULE_TYPE_DENY,
+                'role' => AclConstants::ROOT_ROLE,
+            ];
+        }
+        return $installerRules;
+    }
+}
+```
+
+## 3) Set up behavior
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN                                                                 | DESCRIPTION                                                                                                 | PREREQUISITES | NAMESPACE                                                                                                |
+|------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|---------------|----------------------------------------------------------------------------------------------------------|
+| ServiceProductOfferFormExpanderPlugin                                  | Expands `ProductOfferForm` with `Service points` and `Service` fields.                                           |               | Spryker\Zed\ProductOfferServicePointMerchantPortalGui\Communication\Plugin\ProductOfferMerchantPortalGui |
+| ServiceProductOfferFormViewExpanderPlugin                              | Expands the `ProductOfferForm` Twig template with the `service` form section.                                       |               | Spryker\Zed\ProductOfferServicePointMerchantPortalGui\Communication\Plugin\ProductOfferMerchantPortalGui |
```

Comment:
Is `service` a direct reference to the code?

Reply:
No, replaced with the real section name from the code